mlflow/server/js/src/components/ArtifactView.js

```diff
@@ -288,7 +288,7 @@ decorators.Header = ({style, node}) => {
 decorators.Loading = ({style}) => {
   return (
     <div style={style}>
-      <img className="loading-spinner" src={spinner}/>
+      <img alt="" className="loading-spinner" src={spinner}/>
```

Comment:
Does it make sense to actually have some text here?

Reply:
Not really, it's a decorative spinner. The most useful alt-text would be 'loading' but that's unnecessary because the next word already is 'loading'.